source/backend/api/Areas/Tools/Controllers/LtsaController.cs

```diff
@@ -130,6 +141,59 @@ public async Task<IActionResult> PostLtsaFields(string pid)
             var result = await _ltsaService.PostLtsaFields(pid);
             return new JsonResult(result);
         }
+
+        /// <summary>
+        /// Upload a newline delimited list of pid numbers reflecting BCTFA ownership as per LTSA.
+        /// </summary>
+        /// <param name="ownershipFile">the file containing the list of pids.</param>
+        /// <returns>The orders created within LTSA.</returns>
+        [HttpPut("ownership")]
+        [Produces("application/json")]
+        [ProducesResponseType(200)]
+        [ProducesResponseType(typeof(Pims.Api.Models.ErrorResponseModel), 400)]
+        [SwaggerOperation(Tags = new[] { "tools-ltsa" })]
+        [HasPermission(Permissions.LtsaOwnershipEdit)]
+        public async Task<IActionResult> PutBctfaOwnership(IFormFile ownershipFile)
+        {
+            _logger.LogInformation(
+                "Request received by Controller: {Controller}, Action: {ControllerAction}, User: {User}, DateTime: {DateTime}",
+                nameof(DispositionController),
+                nameof(LtsaController),
+                _user.GetUsername(),
+                DateTime.Now);
+
+            _logger.LogDebug(ownershipFile.Serialize());
+
+            // TODO: placeholder endpoint to facilitate LTSA integration.
+            return new OkResult();
```

Comment:
For now I just want an endpoint that accepts the correct data, and authenticates/authorizes an LTSA service account. Logic can wait until they have confirmed this format works for them.

Reply:
My only comment here would be that we are collocating "two concerns" of the application/API. One concerns deals with "us" talking to their (LTSA) service, and the other concern (the new code added) is allowing "them" to talk to us.
I think I would prefer to separate the two concerns into their own controllers + services. The reason for me is that it looks simple now but it the future this controller might grow and the "in" functionality will get mixed with the "out" functions and will be hard to determine which is which.

I understand it is all "about" LTSA but generically this endpoint could be called by "anyone" with the right claims. What do you think? I'd prefer something like a new controller **LtsaDataIngestionController**  (not set on naming, can be anything else)